CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+## Table of Contents
+
+- [Contributing to Pluto](#contributing-to-pluto)
+- [Code of Conduct](#code-of-conduct)
+- [Ways to contribute](#ways-to-contribute)
+- [Bounties](#bounties)
+- [Submitting a bug report](#submitting-a-bug-report)
+- [Submitting a feature request](#submitting-a-feature-request)
+- [Resolving an issue](#resolving-an-issue)
+  - [Adding tests](#adding-tests)
+  - [Commits](#commits)
+  - [Discuss and update](#discuss-and-update)
+  - [Be aware of the person behind the code](#be-aware-of-the-person-behind-the-code)
+  - [Abandoned or stale pull requests](#abandoned-or-stale-pull-requests)
+  - [Contributions related to spelling and grammar](#contributions-related-to-spelling-and-grammar)
+
+
+
+## Contributing to Pluto
+
+Thank you for your interest in improving Pluto.
+
+There are multiple opportunities to contribute to Pluto, at any level. It doesn't matter if you are just getting started with applied cryptography or if you are an expert; we can use your help. No contribution is too small and we value all contributions.
+
+This document will help you get started. It should be considered as a guide to help you navigate the contribution process. The [Pluto telegram](https://t.me/pluto_xyz/1) is available to help you with any questions concerns that you may have that are not covered in this guide.
+
+If you contribute to this project, your contributions will be made to the project under the Apache 2.0 license.
```

Comment:
link the apache 2.0 license

Reply:
Added the license as a file to this repo and to all other public repos